tools/generate_providers/bin/generate_providers.dart

```diff
@@ -736,7 +736,8 @@ import 'internals.dart';
   builder.writeAll(generateAll(matrix), '\n');
 
   await file.writeAsString(
-    DartFormatter().format(builder.toString()),
+    DartFormatter(languageVersion: DartFormatter.latestLanguageVersion)
+        .format(builder.toString()),
```

Comment:
I'm not sure what this script is for, so I'm not sure if `DartFormatter.latestLanguageVersion` is right here.